src/System.Threading/tests/ReaderWriterLockSlimTests.cs

```diff
@@ -383,6 +383,7 @@ public static void WriterToUpgradeableReaderChain()
 
         [Fact]
         [OuterLoop]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "dotnet/corefx #17443")] // Hangs in desktop
```

Comment:
If I'm right in my previous comment, it would probably be better to mention the issue/PR number that this was added as a regression test for.